doc/documentation.tex

```diff
@@ -466,6 +417,105 @@ \subsubsection{Methods comparison}
 As we can see the execution times differ vastly depending on method and execution policy. We are satisfied with the results especially for Iterative Closest Point and Non-iterative Closest Point methods because we managed to significantly improve the performance by utilizing CUDA library. Therefore, these two methods would be the best choice if time is the most important factor while utilizing SLAM. Coherent Point Drift was expected to be the slowest of the all methods due to its sophistication. However its CPU implementation using Fast Gauss Transform occurred to be faster than ICP CPU variant and it is usable even for clouds with considerable sizes.
 
 \subsection{Noise and outliers tolerance}
+We tested our methods using the clouds and conditions that may be closer to reality. That is, we used clouds that are missing points or even entire part and we corrupted clouds by noise and outliers. For this purpose we have used CPU versions of algorithms. For ICP and NICP \textit{policy} was set to \textit{parallel} and \textit{approximation-type} to \textit{none}. For CPD \textit{approximation-type} was set to \textit{hybrid}, mainly to achieve faster computation times. Many tests were conducted and it turned out that results are highly dependent on configuration. Therefore, we will show the most interesting results and tests that have shown the most consistent characteristics. All results are available in spreadsheet kept on Github: \href{https://github.com/Sliwson/cuda-slam/tree/cpd-documentation/doc/noise}{https://github.com/Sliwson/cuda-slam/tree/cpd-documentation/doc/noise}. Also in directory \textit{config} there are all test configurations.\\
```

Comment:
- points or even entire part -> points, even large part of them, 
- we will show you the -> we present you with the
- last sentence maybe like "All the test configurations are to be found in directory \textit{config} of the github repository".

Reply:
Done